### DISCUSSION TITLE
How do I make an inventory that is common to the entire server?

### DISCUSSION THREAD

**Discussion by TechPro424** [FabricMC/fabric-api#3037] [Mod Dev Support] [answered]
How do I make an inventory that is common to the entire server?
(I should be able to access it in any server-side code)

**Comment by warjort** [comment]
There's not really such a thing as Server data. 
There is data per level/dimension.

You can kind of have server data by always storing that data against the overworld.

See: https://fabricmc.net/wiki/tutorial:persistent_states
or look at how FilledMapItem works.

**Reply by TechPro424** [reply, marked as answer]
I'm using https://fabricmc.net/wiki/tutorial:persistent_states, thanks for your help!
